### ISSUE TITLE
Geta.Optimizely.Categories.Extensions.IEnumerableExtensions.MemberOfAll() or MemberAny() gives build error.

### ISSUE THREAD

**Issue by Deepa-Puranik** [Geta/geta-optimizely-categories#15]
As IEnumerableExtensions using internal access modifier, as its been removed access from this package. If we want to utilize these how to access these methods? Is it no longer needed?

**Comment by brianweet** [comment]
These extensions were not strictly in the scope of this package (they're extensions for content references) and therefore I've decided to make them internal only.
It's a breaking change I failed to document.

I understand that the methods are useful, therefore I've added them to our Geta.Optimizely.Extensions package:
https://github.com/Geta/geta-optimizely-extensions/releases/tag/v2.1.0
The nuget package has been uploaded to the opti nuget feed and should be available shortly.

Another option is to copy the [extension methods](https://github.com/Geta/geta-optimizely-categories/blob/master/src/Geta.Optimizely.Categories/Extensions/IEnumerableExtensions.cs#L10) into your own solution.
